setup/requirements.md

```diff
@@ -43,6 +43,20 @@ currently require the following versions _or newer_:
 If you're using [Remote IDEs](../environments/editors.md), allow pop-ups; Coder
 launches the Remote IDE in a pop-up window.
 
+## Network Policies
+
+Coder uses
+[Container Network Interface (CNI)](https://github.com/containernetworking/cni#what-is-cni)
+plugins to implement network segmentation and tenant isolation in the Kubernetes
+cluster. This enforces network boundaries between pods, preventing users from
+accessing other workspaces.
+
+If there is no CNI plugin configured to enforce such policies, workspaces will
```

Comment:
I think that CNI plugins are required for Kubernetes to work (these interfaces are just gRPC interfaces that separate what Kubernetes is trying to do from the implementation), but not all CNI plugins support enforcing network policies. I think this should be in an aside or with a warning symbol because this is pretty important for production installs

```suggestion
If your Container Network Interface (CNI) plugin does not support NetworkPolicy enforcement, traffic between workspaces and other containerized workloads within the same cluster will be permitted to communicate without restriction. Consider testing your container networking after installation to ensure that the behavior is as you expect.
```

Reply:
Good distinction here - I was not aware of this previously. We should include this @khorne3.

Reply:
Actually, on second thought, maybe we could be more vague and say "If your cluster networknig configuration does not enforce Network Policies ..."

Also, I suggest linking to this page instead of the CNI spec open source repo (that one is more useful for implementers of CNI plugins): https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/